google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/spi/v2/HttpBigQueryRpc.java

```diff
@@ -130,12 +130,23 @@ private void validateRPC() throws BigQueryException, IOException {
   public Dataset getDataset(String projectId, String datasetId, Map<Option, ?> options) {
     try {
       validateRPC();
-      return bigquery
-          .datasets()
-          .get(projectId, datasetId)
-          .setFields(Option.FIELDS.getString(options))
-          .setPrettyPrint(false)
-          .execute();
+      Integer accessPolicyVersion = null;
```

Comment:
Nit. Move declaration of accessPolicyVersion closer to usage.
E.g.,
```
      validateRPC();
      Bigquery.Datasets.Get bqGetRequest =
          bigquery
              .datasets()
              .get(projectId, datasetId)
              .setFields(Option.FIELDS.getString(options))
              .setPrettyPrint(false);
      for (Map.Entry<Option, ?> entry : options.entrySet()) {
        if (entry.getKey() == Option.ACCESS_POLICY_VERSION && entry.getValue() != null) {
          bqGetRequest.setAccessPolicyVersion((Integer) entry.getValue());
        }
      }
      return bqGetRequest.execute();
```

Reply:
Fixed.